src/client/components/pages/admin-panel-search.tsx

```diff
@@ -70,15 +69,17 @@ class AdminPanelSearchPage extends React.Component<Props, State> {
 
 		this.state = {
 			query: props.query,
-			results: props.initialResults,
-			updateResultsTrigger: 0
+			results: props.initialResults
 		};
 
 		this.paginationUrl = './admin-panel/search';
+		this.pagerRef = React.createRef();
 	}
 
 	paginationUrl: string;
 
+	pagerRef: any;
```

Comment:
The refs can be a bit tricky to access (ref.current can potentially be undefined) so it's best to type these strongly to avoid any runtime errors. This should do it:
```
    this.pagerRef = React.createRef<PagerElement>();
    [...]
	pagerRef: React.RefObject<PagerElement>;
```